src/newgrf_industries.cpp

```diff
@@ -605,6 +605,16 @@ void IndustryProductionCallback(Industry *ind, int reason)
 		if (tgroup == NULL || tgroup->type != SGT_INDUSTRY_PRODUCTION) break;
 		const IndustryProductionSpriteGroup *group = (const IndustryProductionSpriteGroup *)tgroup;
 
+		if (group->version == 0xFF) {
+			/* display error message */
+			SetDParamStr(0, spec->grf_prop.grffile->filename);
+			SetDParam(1, spec->name);
+			ShowErrorMessage(STR_NEWGRF_BUGGY, STR_NEWGRF_BUGGY_INVALID_CARGO_PRODUCTION_CALLBACK, WL_WARNING);
```

Comment:
Would it be possible to include the coordinates of the specific industry in the error?